packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -37,14 +43,21 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
 
   protected suggestionStore: SuggestionsController<T>;
   protected SuggestionOfProperType = Suggestions as new (props: ISuggestionsProps<T>) => Suggestions<T>;
-  protected loadingTimer: number | undefined;
   protected currentPromise: PromiseLike<any> | undefined;
+  protected _ariaMap: { [key: number]: string };
+  private _id: string;
 
   constructor(basePickerProps: P) {
     super(basePickerProps);
 
     const items: T[] = basePickerProps.selectedItems || basePickerProps.defaultSelectedItems || [];
 
+    this._id = getId();
+    this._ariaMap = {
```

Comment:
>_ariaMap [](start = 9, length = 8)

Why not just have an interface for Aria map and avoid the use of enum entirely (again, based on my squinting and getting the impression this enum is entirely internal)

Reply:
I waffled about this. I think I chose wrong, I'll go back and make it an interface.